### PR TITLE
Issue201 - just move a func that's only used in one spot

### PR DESCRIPTION
**Pull request by duglin**
See issue #201 for history - just seems unnecessary.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by arschles** [comment]
@duglin can you put "Closes #201" in the PR description, so that issue is automatically closed when this is merged.

Otherwise, LGTM

File: pkg/controller/handler.go [hunk 1]
```diff
@@ -245,3 +243,34 @@ func (h *handler) CreateServiceBroker(in *servicecatalog.Broker) (*servicecatalo
 	glog.Infof("Updating Service Broker %s with State\n%v", in.Name, in.Status.Conditions[0].Type)
 	return h.storage.Brokers().Update(in)
 }
+
+// ConvertCatalog converts a service broker catalog into an array of ServiceClasses
+func ConvertCatalog(in *brokerapi.Catalog) ([]*servicecatalog.ServiceClass, error) {
```
Comment (mkibbe1993) [inline pkg/controller/handler.go]: Can you make ConvertCatalog unexported?

Comment (duglin) [inline pkg/controller/handler.go]: ah good point - done

**Comment by duglin** [comment]
@arschles done - thanks for the reminder

**Comment by MHBauer** [comment]
okay.

**Comment by mkibbe1993** [comment]
LGTM, merging

**Pull request merged**